webpack.config.js

```diff
@@ -13,35 +13,48 @@ module.exports = {
         filename: "main.js"
     },
     module: {
-      rules: [
-        {
-          test: /\.js$/,
-          exclude: /node_modules/,
-          use: [{
-            loader: "babel-loader",
-            options: {
-                "presets": ["@babel/preset-env", "@babel/preset-react"]
-            }
-          }]
-        },
-        {
-            test: /\.css$/,
-            use: [
-              {
-                loader: "style-loader"
-              },
-              {
-                loader: "css-loader",
-                options: {
-                  modules: true,
-                  localIdentName: "[name]_[local]_[hash:base64]",
-                  sourceMap: true,
-                  minimize: true
+        rules: [
+            {
+                test: /\.js$/,
+                exclude: /node_modules/,
+                use: [{
+                    loader: "babel-loader",
+                    options: {
+                        "presets": ["@babel/preset-env", "@babel/preset-react"]
+                    }
+                }]
+            },
+            {
+                test: /\.css$/,
+                use: [
+                {
+                    loader: "style-loader"
+                },
+                {
+                    loader: "css-loader",
+                    options: {
+                    modules: true,
+                    localIdentName: "[name]_[local]_[hash:base64]",
+                    sourceMap: true,
+                    minimize: true
+                    }
                 }
-              }
-            ]
-          }
-      ]
+                ]
+            },
+            {
+                test: /\.(gif|png|jpe?g|svg)$/i,
+                use: [
+                    'file-loader',
+                    {
+                      loader: 'image-webpack-loader',
```

Comment:
You might not need image-webpack-loader.

Reply:
It turns out that I need it.